File: yangxj96-starter/yangxj96-starter-dubbo/src/main/java/com/yangxj96/saas/starter/dubbo/dubbo/auth/RoleDubboService.java
package com.yangxj96.saas.starter.dubbo.dubbo.auth;

import com.yangxj96.saas.bean.user.Authority;

import java.util.List;


public interface RoleDubboService {

    /**
     * 根据角色获取关联的权限
     *
     * @param roleCode 角色CODE
     */
    List<Authority> getAuthorityByRoleCode(String roleCode);

}

File: yangxj96-starter/yangxj96-starter-dubbo/src/main/kotlin/com/yangxj96/saas/starter/dubbo/dubbo/auth/RoleDubboService.kt
package com.yangxj96.saas.starter.dubbo.dubbo.auth

import com.yangxj96.saas.bean.user.Authority


interface RoleDubboService {

    /**
     * 根据角色获取关联的权限
     *
     * @param roleCode 角色CODE
     */
    fun getAuthorityByRoleCode(roleCode: String): List<Authority>

}